web/layout_attributes/using_flex/my_element.dart

```diff
@@ -0,0 +1,13 @@
+@HtmlImport('my_element.html')
+library my_element;
+
+import 'package:web_components/web_components.dart' show HtmlImport;
+import 'package:polymer/polymer.dart';
+import 'package:polymer_elements/paper_styles.dart';
+
+/// Silence analyzer [JsArray]
```

Comment:
Was this for `paper_styles.dart`? You can add a prefix to it and then silence it that way:

```dart
import 'package:polymer_elements/paper_styles.dart' as paper_styles;

/// Uses [paper_styles]
```